src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import './styles/_calculator.scss';
+import Calculator from './components/calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] You don't need to remove the `App` component even though you have only a single child component `Calculator`. For the sake of `React/JS best practice,` you should always use the `App` component as a wrapper. Kindly brig back `App.js` and render the `Calculator` component inside it. :pray: